app/templates/views/accessibility_statement.html

```diff
@@ -113,7 +113,7 @@ <h3 class="heading-small">Non compliance with the accessibility regulations</h3>
 
   <ul class="govuk-list govuk-list--bullet">
     <li>
-      the (before <abbr title="HyperText Markup Language">HTML</abbr>) code for the ‘upload a file’ button may cause issues for assistive technology users - this fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships">success criterion 1.3.1: info and relationships</a>
+      the code for the ‘upload a file’ button may cause issues for assistive technology users - this fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships">success criterion 1.3.1: info and relationships</a>
```

Comment:
@tombye reintroducing the abbreviation tag, just in case.

```suggestion
      the <abbr title="HyperText Markup Language">HTML</abbr> code for the ‘upload a file’ button may cause issues for assistive technology users - this fails <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships">success criterion 1.3.1: info and relationships</a>
```